nimbus-ui/nimbusui/src/app/components/platform/card/card-details-field.component.html

```diff
@@ -21,13 +20,12 @@
          </ng-template>
 
         <div class="form-group" *ngIf="!element.config.uiStyles.attributes.inplaceEdit && !isDate(element.config.type.name)">
-           <!-- <label class="" *ngIf="element.config.uiStyles.attributes.showName==true">{{label}}</label> -->
            <nm-input-label *ngIf="element.config.uiStyles.attributes.showName==true"
                 [element]="element" 
                 [required]="false">
             </nm-input-label>
             <div class="textWrapBreakWord" [nmDisplayValue]='element.leafState' [config]='element.config'>
-                <span class="fieldValue">{{value ? value : placeholder}}&nbsp;</span>
+                <span class="fieldValue">{{displayValue}}&nbsp;</span>
```

Comment:
Would be better if this is transformed using a directive

Reply:
@sandeepmantha Fixed in latest commit... will you please review?